.github/workflows/main.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: /tmp/_posts
-          key: ${{ hashFiles('.github/pandoc-version') }}-${{ hashFiles('.github/R-version') }}-${{ hashFiles('.github/pkg-versions.Rds') }}-${{ hashFiles('.github/pkg-deps.Rds') }}
+          key: RStudio-AI-Blog-${{ hashFiles('.github/pandoc-version') }}-${{ hashFiles('.github/R-version') }}-${{ hashFiles('.github/pkg-versions.Rds') }}-${{ hashFiles('.github/pkg-deps.Rds') }}
```

Comment:
Per discussion, we can try modifying the cache key for build artifacts (which is different from cache key for R packages BTW). This will cause all blog posts to be built from scratch and we can then see whether that fixes the RSS issue.